samples/react-native/src/App.tsx

```diff
@@ -57,6 +57,10 @@ Sentry.init({
     logWithoutTracing('Transaction beforeSend:', event.event_id);
     return event;
   },
+  beforeSendMetric(metric: Sentry.Metric) {
+    logWithoutTracing('Metric beforeSend:', metric.name, metric.value);
+    return metric;
+  },
```

Comment:
Wdyt of enabling metrics for the sample app and adding an example in the app (e.g. a button or screen that uses metrics)

```suggestion
  },
  enableMetrics: true,
```